roles/aodh/defaults/main.yml

```diff
@@ -18,7 +18,7 @@ aodh:
   source:
     rev: 'stable/newton'
     constrain: True
```

Comment:
Get rid of `constrain: True` we should always use contrain and not provide an option to turn it off.